packages/input-select/README.md

```diff
@@ -0,0 +1,56 @@
+# Farmblocks Select Input
+
+A component for rendering Select inputs
+
+## Installation
+
+```
+npm install @crave/farmblocks-input-select
+```
+
+## Usage
+
+```
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+import Select from '@crave/farmblocks-input-select';
+
+const items = [
+  { value: "1", label: "Apple" },
+  { value: "2", label: "Banana" },
+  { value: "3", label: "Pear" }
+];
+
+class App extends Component {
+  render() {
+    return (
+      <Select
+        placeholder="Select fruit"
+        label="Fruit"
+        items={items}
+        onChange={action("onChange")}
```

Comment:
copy/paste of this wont work because of action()

Reply:
Opsss